db/schema.rb

```diff
@@ -11,7 +11,17 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170411191927) do
+ActiveRecord::Schema.define(version: 20170630141235) do
+
+  create_table "bulk_update_searches", force: :cascade do |t|
```

Comment:
seems odd that this change made it into the PR. also, are we planning on storing these at some point?